src/app/dev/DevToys.Tools/Tools/Converters/JsonTable/JsonTableConverterGuiTool.cs

```diff
@@ -177,17 +139,39 @@ private async Task ConvertAsync(ConvertTarget target, CancellationToken cancella
                     case ConvertTarget.Clipboard:
                         await _clipboard.SetClipboardTextAsync(conversionResult.Text);
                         break;
+
+                    case ConvertTarget.Save:
+                        Stream? fileStream = null;
+                        try
+                        {
+                            fileStream = await _fileStorage.PickSaveFileAsync("txt", "csv");
```

Comment:
I think this can be slightly simplified as 
`{ using Stream? fileStream = ... }`
ie without the try-finally. It doesn't try to dispose a null fileStream when the save dialog is cancelled.

Reply:
Good catch, thank you !